test/stdlib/StringBridge.swift

```diff
@@ -77,6 +77,38 @@ StringBridgeTests.test("Constant NSString New SPI") {
   }
 }
 
+StringBridgeTests.test("Constant NSString New SPI") {
+  if #available(SwiftStdlib 6.2, *) {
```

Comment:
Nate added a modifier to require a particular stdlib version: https://github.com/swiftlang/swift/pull/79838/files
This way the test is listed as skipped when compiled for an older target.